src/vumi2/applications/turn_channels_api/messages.py

```diff
@@ -38,15 +38,15 @@ def turn_inbound_from_msg(message: Message, channel_id: str) -> dict:
     """
     msg = {
         "contact": {
-            "id": message.to_addr,
-            "profile": {"name": message.to_addr},
+            "id": message.from_addr,
+            "profile": {"name": message.from_addr},
         },
         "message": {
             "type": "text",
             "text": {
                 "body": message.content,
             },
-            "from": message.from_addr,
+            "from": message.to_addr,
```

Comment:
I think this must stay `from_addr`. `String. The user ID. A Channel can respond to a user using this ID.`

I don't think Turn wants our `to_addr` at all (we give that when we set up the channel)